sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -640,6 +642,9 @@ static void _handle_rtr_adv(gnrc_netif_t *netif, const ipv6_hdr_t *ipv6,
     uint32_t next_timeout = UINT32_MAX;
 
     assert(netif != NULL);
+    if (gnrc_netif_is_6lbr(netif)) {
+        return; /* this should not happen */
+    }
```

Comment:
This can happen still with router advertisements that do *not* propagate a default route (e.g. the `last_ra` ones send with `gnrc_ipv6_auto_subnets` that contain the Route Information Option)